### PR TITLE
Add custom domain to riseprotocol.eth

### PR DESCRIPTION
**Pull request by RiseProtocol**
#### Hi! What is your PR about?

- [] Add images for your space
- [X ] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -130,4 +130,5 @@
   "poll.santiment.net": "santiment.eth",
   "voting-bmxx.multiplier.finance": "mcl-multiplierfinance.eth",
   "vote.cndy.finance": "candyfinance.eth"
+  "riseprotocol.io/governance': "riseprotocol.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:133]: A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.cndy.finance": "candyfinance.eth",
  "riseprotocol.io/governance': "riseprotocol.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by RiseProtocol — COMMENTED** [review]
Added comma. 
Changed custom domain to "gov.riseprotocol.io"

File: spaces/domains.json [hunk 2]
```diff
@@ -129,5 +129,6 @@
   "vote.1-b.tc":"1btc.eth",
   "poll.santiment.net": "santiment.eth",
   "voting-bmxx.multiplier.finance": "mcl-multiplierfinance.eth",
-  "vote.cndy.finance": "candyfinance.eth"
+  "vote.cndy.finance": "candyfinance.eth",
+  "gov.riseprotocol.io': "riseprotocol.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:133]: You must replace the single quote with a double quote.
```suggestion
  "gov.riseprotocol.io": "riseprotocol.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by RiseProtocol — COMMENTED** [review]
Edited to include double quotes.

**Comment by RiseProtocol** [comment]
Please review my changes and merge the branch. Thanks!

**Review by gokaiorg — APPROVED** [review]

**Comment by gokaiorg** [comment]
The `domains.json` file has been updated since your last commit and there is a conflict in your PR, can you resolve it?

**Review by RiseProtocol — COMMENTED** [review]
Copied over latest domain.json file and added custom riseprotocol.eth domain.

**Comment by gokaiorg** [comment]
There is still a conflict, can you try the following commands? Thank you
`git remote add master https://github.com/snapshot-labs/snapshot-spaces.git`
`git fetch master`
`git checkout master`
`git merge master/master`

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**